notesserver/settings/common.py

```diff
@@ -38,9 +38,11 @@
 ROOT_URLCONF = 'notesserver.urls'
 
 MIDDLEWARE_CLASSES = (
+    'edx_django_utils.cache.middleware.RequestCacheMiddleware',
     'corsheaders.middleware.CorsMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
+    'edx_rest_framework_extensions.middleware.RequestMetricsMiddleware',
```

Comment:
Reviewer FYI: Since there is no Session Middleware this was failing with the TieredCacheMiddleware, so I dropped it since that is only used for testing.